tgui/packages/tgui/styles/interfaces/WirePanel.scss

```diff
@@ -0,0 +1,19 @@
+@use '../colors.scss';
+@use '../components/Button.scss';
+@use '../base.scss';
+
+@keyframes fader {
+  0% {
+    opacity: 1;
+  }
+  50% {
+    opacity: 0.5;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+.WirePanel-silicon-disabled {
+  animation: fader 2s linear 0 infinite;
+}
```

Comment:
Nice little visual that isn't necessary to tie to wire panels. Might be worth putting this in some kind of shared animations stylesheet, so that other people who want to do the same can just bring it in rather than writing their own.